CHANGELOG.md

```diff
@@ -1,5 +1,70 @@
 # Node.js Agent for Google Cloud Trace ChangeLog
 
+## 2017-02-28, Version 1.0.0-rc1, @matthewloring
+
+### Notable changes
+
+This version introduces a new API for creating custom trace spans as well as an API for writing custom plugins
```

Comment:
I've expanded this to say that it is semver major and that the changes should primarily affect users of the custom tracing API. The full api surface has changed so I'm not sure a list of changes makes sense. What level of detail did you have in mind?